sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ArtifactManifestProperties.cs

```diff
@@ -9,24 +9,24 @@
 namespace Azure.Containers.ContainerRegistry
 {
     /// <summary> Manifest attributes details. </summary>
-    public partial class RegistryArtifactProperties
+    public partial class ArtifactManifestProperties
     {
-        internal RegistryArtifactProperties(string digest, string architecture, string operatingSystem)
+        internal ArtifactManifestProperties(string digest, string architecture, string operatingSystem)
         {
             this.Digest = digest;
-            this.CpuArchitecture = architecture;
+            this.Architecture = architecture;
             this.OperatingSystem = operatingSystem;
         }
 
         /// <summary> List of manifest attributes details. </summary>
         internal IReadOnlyList<ManifestAttributesManifestReferences> References { get; }
 
         /// <summary></summary>
-        public IReadOnlyList<RegistryArtifactProperties> RegistryArtifacts
+        public IReadOnlyList<ArtifactManifestProperties> Manifests
         {
             get
             {
-                List<RegistryArtifactProperties> artifacts = new List<RegistryArtifactProperties>();
+                List<ArtifactManifestProperties> artifacts = new List<ArtifactManifestProperties>();
```

Comment:
Same here about capacity (I'll not repeat).

Reply:
Thanks, I'll find all places I can change this.